pkgs/development/python-modules/bce-python-sdk/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, future
+, pycryptodome
+, six
+}:
+
+buildPythonPackage rec {
+  pname = "bce-python-sdk";
+  version = "0.8.87";
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "vgTf1HiD/R/1m24zq/vB+Dt2KzacfwePrfe1q0jB0qk=";
+  };
+  buildInputs = [];
+  checkInputs = [];
+  nativeBuildInputs = [];
+  propagatedBuildInputs = [
+    future
+    pycryptodome
+    six
+  ];
+
+  meta = with lib; {
+    description = "Baidu Cloud Engine Python SDK";
+    homepage = "http://bce.baidu.com/";
+    license = licenses.asl20;
+  };
```

Comment:
This package is a dependency of PaddleOCR, and may never be used by anything else. It has no documentation, and seems to be something Baidu-specific. Should I package it like this, or just nest it inside the PaddleOCR package?

Reply:
adding it as a separate package is fine IMO.